src/workshop.md

```diff
@@ -23,24 +16,24 @@ directions for the project.
 **For questions, please contact the meeting organizers at**
 **[mfem@llnl.gov](mailto:mfem@llnl.gov).**
 
-<!--
 ### Registration
 
-Registration closed on October 11th.
+If you plan to attend, [please register no later than **October 17th**](https://docs.google.com/forms/d/e/1FAIpQLSf_YKPpCuzXjK2ZeX-lJzPsgd6Cfdv3UCOzWtPUiwgPoXzhMA/viewform?usp=sf_link).
+
+There is no registration fee. Zoom details will be distributed to participants prior to the event date.
 
 ### Meeting format
 
 Depending on the interest and user feedback, the meeting will include the following elements:
 
 - Project news and development updates from the MFEM team
-- An overview of the latest features in MFEM-4.4 and GLVis-4.2
+- An overview of the latest features in MFEM-4.5, MFEM-4.5.2 and MFEM-4.6
 - Contributed talks from application developers utilizing MFEM
 - Roadmap discussion for future development
 - Technical discussions in breakout rooms for Electromagnetics, Fluids, and
   Structural Mechanics applications
 
-See also the agenda for the previous [2021](../workshop21) MFEM workshop.
--->
+See also the agenda for the previous [2022](../workshop22), [2021](../workshop21) MFEM workshops.
 
 ### Meeting format
```

Comment:
Perhaps we could rename this section `Agenda` and move the note about slack to the `Meeting format` section above.